js_modules/dagit/packages/core/src/instance/InstanceConfig.tsx

```diff
@@ -96,56 +61,15 @@ export const InstanceConfig = React.memo(() => {
           Dagster version: <Code style={{fontSize: '16px'}}>{data.version}</Code>
         </Subheading>
       </Box>
-      <YamlShimStyle />
-      <Box padding={{vertical: 16, horizontal: 24}}>
-        {sections.map((section) => {
-          const [id] = section.split(/\:/);
-          const hashForSection = `#${id}`;
```

Comment:
Yeah, I noted it in the summary. I'm not sure how important that feature has been. `scrollToLine` sounds like it might be a good approach -- I'll see if I can make that work.